jenkinsfile-runner-base-image/packager-config.yml

```diff
@@ -237,17 +238,17 @@ plugins:
   - groupId: "org.jenkins-ci.plugins"
     artifactId: "git"
     source:
-      version: "4.0.0"
+      version: "4.1.1"
 # Pipeline: SCM Step (Jenkins-Version: 2.60)
   - groupId: "org.jenkins-ci.plugins.workflow"
     artifactId: "workflow-scm-step"
     source:
-      version: "2.9"
+      version: "2.10"
```

Comment:
I don't know much about the rest, but this I want. I'll let the others approve, however.

Reply:
FYI: We should update everything regularly. And when we want to update a single version we have to ensure that dependencies and transitive dependencies are also installed and in the right version.